### ISSUE TITLE
aws_cognito_user_pool checks

### ISSUE THREAD

**Issue by hs-amartin** [terraform-linters/tflint-ruleset-aws#147]
If you don't specify an email config, the `aws_cognito_user_pool_invalid_email_verification_message` check errors out on null reference

```
resource "aws_cognito_user_pool" "this" {
  count = var.create ? 0 : 1
  name  = "Foo"
}
```

Error:
```
Error: Failed to check `aws_cognito_user_pool_invalid_email_verification_message` rule: Failed to eval an expression in main.tf:##; Null condition: The condition value is null. Conditions must either be true or false.
```

**Comment by wata727** [comment]
Did you explicitly pass null to the value of `var.create`? If you pass an unknown value instead, TFLint will handle it correctly:

```hcl
# unknown value
variable "create" {}

resource "aws_cognito_user_pool" "this" {
  count = var.create ? 0 : 1 # no errors
  name  = "Foo"
}
```

```hcl
# null value
variable "create" {
  default = null
}

resource "aws_cognito_user_pool" "this" {
  count = var.create ? 0 : 1 # Null condition: The condition value is null. Conditions must either be true or false.
  name  = "Foo"
}
```

If for some reason you need to pass a null value, please let us know your use case.

**Comment by hs-amartin** [comment]
@wata727 Sorry, that is transparently obvious in my simplified example (I have a more complex conditional in my actual use case, but the case where the value resolves to null doesn't need to be supported/was a mistake in my original).
